Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/UnityUIInteractables.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: 125b603c788cf5c43ba792f00c2ac9d3
```

Comment:
Also simplify to UnityUI, Interactable is redundant as it's already a child folder of the path

Reply:
> Also simplify to UnityUI, Interactable is redundant as it's already a child folder of the path

Just naming the folder UnityUI feels odd. I think it is okay to call it UnityUIInteractables. Calling it UnityUIButtons could be misleading for some people, since these are using the interactable script to deliver a button like experience instead of Unity UI Button script.

Reply:
Why is it odd? It's already under the Interactables folder, why do we need to mention Interactables again?
Look at Interactable/Prefabs/UnityUI/{my stuff}
vs Interactable/Prefabs/UnityUIInteractables/{my stuff}

As a user, I know I am under Interactables, how does UnityUIInteractables vs UnityUI give me more information that I don't already have?